src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,31 @@
 package core.basesyntax;
 
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.figurehandlers.FigureSupplier;
 /**
  * Feel free to remove this class and create your own.
  */
+
 public class HelloWorld {
```

Comment:
take a look at the comment :)

you should better remove this class and create your own with a `main` method